lib/ui/theme/text_styles.dart

```diff
@@ -20,7 +20,18 @@ class AppStyles extends TextTheme {
     bodyText2,
     caption,
     overline,
-  });
+  }) : super(
```

Comment:
Good catch, but what about of calling super in each property?
```dart
  AppStyles({
    required this.customStyle,
    super.headline1,
    super.headline2,
    super.headline3,
    super.headline4,
    super.headline5,
    super.headline6,
    super.bodyLarge,
    super.bodyMedium,
    super.bodySmall,
    super.bodyText1,
    super.bodyText2,
    super.caption,
    super.overline,
  });
```

I think it's the same, and you have to write less code

Reply:
the issue here is that there is a mix of material design 2 with 3 maybe i will have to take another look and define the style correctly but if we send all those params to the constructor it will crash

Reply:
With this, there is no conflict, but we use material 3:
```suggestion
  }) : super(
          bodyLarge: bodyLarge,
          displayLarge: displayLarge,
          displayMedium: displayMedium,
          displaySmall: displaySmall,
          headlineMedium: headlineMedium,
          headlineSmall: headlineSmall,
          titleLarge: titleLarge,
          titleMedium: titleMedium,
          titleSmall: titleSmall,
          bodyMedium: bodyMedium,
          bodySmall: bodySmall,
          labelLarge: labelLarge,
          labelSmall: labelSmall,
        );
```

Reply:
Cool Yes, and you can use super en each property too, it should be the same and you have to write le half of the lines